src/app/components/Grid/index.jsx

```diff
@@ -71,13 +71,15 @@ const layoutGridItemSmall = ({ padding = {} }) => `
 
 export const GelPageGrid = styled(Grid)`
   width: 100%;
-  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MAX}) {
-    margin: 0 auto;
-    max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN};
-  }
-  @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
-    margin: 0 auto;
-    max-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN};
+  && {
```

Comment:
what does this do?

Reply:
This is a solution to these rules failing to override the rules in psammead-grid.  The existing code worked in emotion 10, but emotion 11 seems to output the styles in a different order, so psammead is overriding simorgh rather than the other way around

The && increases specificity and ensures these rules have an effect - still trying to find a better solution

Reply:
Changed this to just use `!important`

Been digging into this for a couple of hours now and haven’t come up with a better solution - the problem here is just the order of appearance rule - the styles from `psammead-grid` are now being output after the styles for `GelPageGrid`, so they take precedence

In fact, the `GelPageGrid` is being output in the very first style block: 
![Screen Shot 2021-01-25 at 18 18 05](https://user-images.githubusercontent.com/8601460/105748347-f0a43500-5f39-11eb-929f-44e88fe2841d.png)

The psammead grid (and most other) styles are being output much later - in a different section of the `head`.  That separation is controlled from here: https://github.com/bbc/simorgh/blob/latest/src/server/Document/component.jsx#L58 but I'm not at all clear what causes some styles to be output in that first style element (ie, above the helmet output) and what causes other styles to be output below - any ideas @simonsinclair ?

Either way, those bottom styles set `margin: 0` on the elements we're trying to set `margin: 0 auto` on in this `GelPageGrid`.  And, because both rules have the exact same specificity, the one that occurs later is the one that is used.  In production, there is just one style block, but the specificity is documented as being consistent between modes since Emotion 11.  That does seem to be true in Simorgh too - production build has the same grid issue if I don't bump the specificity here. 

We could start writing logic to control the insertion order in that `DocumentComponent` but, unless more thorough testing shows this to be a wider scale problem, I don’t think it’s worth the hassle.  

Emotion’s default insertion order is just to “insert the styles in the order they are used” - that does seem to be exactly what we want, but there is just something about how this grid is composed that is causing the GelPageGrid styles to be inserted before PsammeadGrid.  I can’t seem to fix that in Simorgh after a couple of hours of experimenting, and I don’t want to have to start making Psammead changes at this point

Reply:
The grid stuff needs a rethink altogether! Happy to go with your approach and maybe have a discussion some time about how to make it better. Seems to cause us a lot of headaches.